src/main/kotlin/com/demonwav/mcdev/i18n/intentions/ConvertToTranslationIntention.kt

```diff
@@ -0,0 +1,88 @@
+/*
+ * Minecraft Dev for IntelliJ
+ *
+ * https://minecraftdev.org
+ *
+ * Copyright (c) 2017 minecraft-dev
+ *
+ * MIT License
+ */
+
+package com.demonwav.mcdev.i18n.intentions
+
+import com.demonwav.mcdev.i18n.I18nElementFactory
+import com.demonwav.mcdev.util.runWriteAction
+import com.intellij.codeInsight.intention.PsiElementBaseIntentionAction
+import com.intellij.lang.java.JavaLanguage
+import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.fileEditor.FileDocumentManager
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.roots.ProjectRootManager
+import com.intellij.openapi.ui.InputValidatorEx
+import com.intellij.openapi.ui.Messages
+import com.intellij.psi.JavaPsiFacade
+import com.intellij.psi.PsiDocumentManager
+import com.intellij.psi.PsiElement
+import com.intellij.psi.PsiLiteral
+import com.intellij.psi.codeStyle.JavaCodeStyleManager
+import com.intellij.util.IncorrectOperationException
+
+class ConvertToTranslationIntention : PsiElementBaseIntentionAction() {
+    @Throws(IncorrectOperationException::class)
+    override fun invoke(project: Project, editor: Editor, element: PsiElement) {
+        if (element.parent is PsiLiteral) {
+            val value = (element.parent as PsiLiteral).value as String?
+            val result = Messages.showInputDialogWithCheckBox("Enter translation key:",
+                "Convert String Literal to Translation",
+                "Replace literal with call to I18n (only works on clients!)",
+                true,
+                true,
+                Messages.getQuestionIcon(),
+                null,
+                object : InputValidatorEx {
+                    override fun getErrorText(inputString: String): String? {
+                        if (inputString.contains('='))
```

Comment:
This check is redundant if the `inputString` is empty so it would make more sense to do the `isEmpty` check first, and then check if the non-empty string contains `=`